spec/requests/api/v1/locations/locations_request_spec.rb

```diff
@@ -27,14 +26,24 @@
       expect(trip[:attributes][:location]).to eq("80030")
     end
 
-    # it 'can return elevation and humidity for a given location (valid)' do
-    #   loc_params = { location: "7" }
-    #   headers = { "CONTENT_TYPE" => "application/json" }
-    #
-    #   post "/api/v1/location", headers: headers, params: JSON.generate(loc_params)
-    #
-    #   expect(response).to_not be_successful
-    #   expect(response.status).to eq(404)
-    # end
+    it 'can return error for invalid location' do
+      loc = "7"
+
+      get "/api/v1/location?location=#{loc}"
```

Comment:
Thank you for the switch from POST to GET. That was a great learning convo for me